__tests__/app.test.js

```diff
@@ -113,3 +113,58 @@ describe('GET /api/reviews/:review_id', () => {
     })
 
 })
+
+describe('6. GET /api/reviews/:review_id/comments', () => {
+    test('status:200 returns the array of comments of the given review_id', () => {
+        return request(app)
+        .get(`/api/reviews/2/comments`)
+        .expect(200)
+        .then(({body}) => {
+            const {comments} = body;
+                expect(comments).toBeInstanceOf(Array);
```

Comment:
we don't need to explicitly check for an array here as it is implicit in using a forEach a bit further down